backend/functions/views.py

```diff
@@ -1,30 +1,178 @@
+        api_type (str): Type of API to be used for translation. (default: indic-trans)
+            Allowed - [indic-trans, google]
+    """
+
+    # Iterate through the languages
+    for output_language in languages:
+
+        # Convert the input_sentences list into a dataframe
+        input_sentences_df = pd.DataFrame(
+            input_sentences,
+            columns=[
+                "sentence_text_id",
+                "corrected_text",
+                "input_language",
+                "context",
+                "quality_status",
+                "metadata",
+            ],
+        )
+
+        # Keep only the clean sentences
+        input_sentences_df = input_sentences_df[
+            (input_sentences_df["quality_status"] == "Clean")
+        ].reset_index(drop=True)
+
+        # Check if the dataframe is empty
+        if input_sentences_df.shape[0] == 0:
+            return {"error": "No clean sentences found. Perform project export first."}
+
+        # Make a sentence list for valid sentences to be translated
+        all_sentences_to_be_translated = input_sentences_df["corrected_text"].tolist()
+
+        # Loop through all the sentences to be translated in batch format
+        for i in range(0, len(all_sentences_to_be_translated), batch_size):
+
+            batch_of_input_sentences = all_sentences_to_be_translated[
+                i : i + batch_size
+            ]
+
+            # Check the API type
+            if api_type == "indic-trans":
+
+                # Get the translation using the Indictrans NMT API
+                translations_output = get_batch_translations_using_indictrans_nmt_api(
+                    batch_of_input_sentences,
+                    input_sentences_df["input_language"].iloc[0],
+                    output_language,
+                )
+
+                # Check if translation output didn't return an error
+                if isinstance(translations_output, Exception):
+                    return translations_output
+
+                # Collect the translated sentences
+                translated_sentences = [
+                    translation["target"] for translation in translations_output
+                ]
+
+            elif api_type == "google":
+
+                # Get the translation using the Indictrans NMT API
+                translations_output = get_batch_translations_using_google_translate(
+                    sentence_list=batch_of_input_sentences,
+                    target_language=output_language,
+                )
+
+                # Check if translation output didn't return an error
+                if isinstance(translations_output, Exception):
+                    return translations_output
+
+                # Collect the translated sentences
+                translated_sentences = [
+                    translation["translatedText"] for translation in translations_output
+                ]
```

Comment:
Same comment as above

Reply:
Alright, I will make that change for both the functions in that case.